core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -673,6 +778,8 @@ <h3 translate="I18N_LEARNER_DASHBOARD_SUGGESTION_TEXT"></h3>
   {{ super() }}
   <script src="{{TEMPLATE_DIR_PREFIX}}/components/background/BackgroundBannerDirective.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/components/summary_tile/CollectionSummaryTileDirective.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/domain/learner_dashboard/LearnerPlaylistService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/library/LearnerDashboardIdsBackendApiService.js"></script>
```

Comment:
This should not go in pages/library. It should go somewhere sensible in domain/. Stuff in pages/ is for just that specific page.

Reply:
Done!